components/support/nimbus-fml/src/frontend.rs

```diff
@@ -251,6 +254,7 @@ impl ManifestFrontEnd {
                         .iter()
                         .map(|v| self.get_prop_def_from_field(v))
                         .collect(),
+                    allow_coenrollment: body.allow_coenrollment.clone(),
```

Comment:
Does this need the `.clone()` here?

Reply:
<img width="1165" alt="image" src="https://github.com/mozilla/application-services/assets/43795363/3530f211-e9bb-46a6-b21b-fb9fa8036cc8">

ahh, thank you clippy 📎

Reply:
Yep the way I think about it is if you're going to move a variable and ALSO use it afterwards, then you'll need to clone it. Otherwise you can just move it and call it a day.